assets/templates/kubernetes.container/configs.yml

```diff
@@ -0,0 +1,56 @@
+fields:
+  - name: cloud.availabilit_zone
+    value: "europe-west1-d"  
+  - name: agent.id
+    value: "12f376ef-5186-4e8b-a175-70f1140a8f30"
+  - name: agent.name
+    value: "kubernetes-scale-123456"
+  - name: agent.ephemeral_id  
+    value: "f94220b0-2ca6-4809-8656-eb478a66c541"
+  - name: agent.version
+    value: "8.7.0"
+  - name: agent.snasphost
+    value: false
+  - name: metricset.period
+    value: 10000
+  - name: event.duration
+    range:
+      min: 1
+      max: 4000000
+  - name: faults
+    range:
+      min: 0
+      max: 500000
+  - name: kubernetes.container.rootfs.inodes.used
+    range:
+      min: 1
+      max: 100000
+  - name: container.network.ingress.bytes
+    range:
+      min: 0
+      max: 15000
+  - name: container.network.egress.bytes
+    range:
+      min: 0
+      max: 10000
+  - name: Offset
+    range:
+      min: 1
+      max: 100
+  - name: Bytes
+    range:
+      min: 1
+      max: 3000000
+  - name: Percentage
+    range:
+      min: 0
+      max: 5000
+    fuzziness: 0.005
+  - name: container.name
+    enum: ["web", "default-http-backend", "dnsmasq", "csi-driver", "web", "web", "web", "prometheus", "konnectivity-agent", "sidecar", "kubedns", "metrics-server-nanny", "web", "web", "fluentbit", "autoscaler", "gke-metrics-agent", "elastic-agent", "web", "kube-state-metrics", "metrics-server", "fluentbit", "elastic-agent", "web", "prometheus-to-sd-exporter"]
+  - name: timedate
+    # Add more dates below in case you want your data to be spread
+    enum: ["2023-05-15" , "2023-05-16"]
```

Comment:
you can replace `timedate` and `timehour` with a single:
```
- name: timestamp
  period: 24h
```

Reply:
Yes I thought of that but at the time of latest tests the PR was not yet merged.

Do you think is a problem to let them like this now?

Reply:
Also I have my dates fixed as the rally track will run some visualisations that have queries in specific timerange (see here https://github.com/elastic/rally-tracks/pull/373/files#diff-18fad8cb6e0990bf9839cad4594f45f8cfde9d2a0a8cce9495614f74c5ab670cR2)

Reply:
if the rally tracks have fixed dates I'd leave it as it is.
it's not a problem per se, indeed it seems it is intended because of the rally tracks, but it's just my personal preference of having as less "hardcoded" stuff as possible :)